README.md

```diff
@@ -1,23 +1,43 @@
 # Minimal Anti-Collusion Infrastructure
 
-[![CI](https://github.com/privacy-scaling-explorations/maci/actions/workflows/e2e.yml/badge.svg)](https://github.com/privacy-scaling-explorations/maci/actions/workflows/e2e.yml)
+[![CI][cli-actions-badge]][cli-actions-link]
+![License](https://img.shields.io/badge/license-MIT-green)
+[![Telegram][telegram-badge]][telegram-link]
+
+
 
 Please refer to
 the original [ethresear.ch
-post](https://ethresear.ch/t/minimal-anti-collusion-infrastructure/5413) for a
+post][telegram-link] for a
```

Comment:
This should like to the blog post, telegram link is down

Reply:
Reflected in https://github.com/privacy-scaling-explorations/maci/pull/566/commits/26d99aed887d950931b5e930d03276481c408b24